src/ifconfig.interfaces.js

```diff
@@ -25,13 +27,22 @@ module.exports = function (cp) {
           return f(stderr);
         }
 
-        f(null, parse(ifConfigOut, routeOut));
+        var ifConfigFileContent = '';
+        try {
+          ifConfigFileContent = fs.readFileSync(interfaces.FILE, {encoding: 'UTF-8'});
```

Comment:
Since we are already in an async context, why not use readFile instead of readFileSync?

Reply:
Yeah that makes more sense. I changed it